tests/tacacs/test_ro_disk.py

```diff
@@ -180,7 +180,11 @@ def test_ro_disk(localhost, ptfhost, duthosts, enum_rand_one_per_hwsku_hostname,
         duthost.copy(src=conf_path, dest="/etc/rsyslog.d/000-ro_disk.conf")
 
         # To get file in decent size. Force a rotate
-        duthost.shell("logrotate --force /etc/logrotate.d/rsyslog")
+        try:
+            duthost.shell("logrotate --force /etc/logrotate.d/rsyslog")
+        except RunAnsibleModuleFail as e:
```

Comment:
Is it possible to further limit the exceptions for safe skipping?
Like checking stderr for known safe error messages.

Reply:
Fixed, only ignore parallel log rotate message.